package.json

```diff
@@ -11,7 +14,7 @@
     "turbo": "^1.13.2"
   },
   "engines": {
-    "node": "18"
+    "node": ">=18"
```

Comment:
I would like to preface this review with saying i'm not a frontend dev, so I might ask some dumb questions. That being said, is there are reason this cannot be locked to "node 18" and needs to allow for ">= 18". Seems it would be best to be precise.